.github/workflows/pr-analysis.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        platform: [ macos-latest, ubuntu-latest, windows-latest ]
+        platform: [ windows-latest ]
     runs-on: ${{ matrix.platform }}
```

Comment:
Why only windows? And if that's the case, remove the matrix and directly set runs-on to windows-latest

Reply:
As platy desktop is meant for building windows container and linux containers there is no need for it to build on ubuntu and macos platforms. It will also speed up development (As we won´t care ubuntu and macos build) and pipeline speed.